snuba/datasets/storages/generic_metrics.py

```diff
@@ -49,7 +49,7 @@ def produce_policy_creator() -> DeadLetterQueuePolicy:
     )
 
 
-aggregated_columns: Sequence[Column[SchemaModifiers]] = [
+common_columns: Sequence[Column[SchemaModifiers]] = [
```

Comment:
I can take out this refactor if people prefer. In that case we'd make this change when adding the storage/consumer, but I didn't want to add too much PR scope